tensorflow/python/keras/engine/saving.py

```diff
@@ -73,8 +66,11 @@ def save_model(model, filepath, overwrite=True, include_optimizer=True):
   Raises:
       ImportError: if h5py is not available.
   """
-
-  if h5py is None:
+  # pylint: disable=g-import-not-at-top
+  try:
+    import h5py
+    HDF5_OBJECT_HEADER_LIMIT = 64512
```

Comment:
HDF5_OBJECT_HEADER_LIMIT used to be a module-wide global variable. Now it is private to this function. Won't this cause problems for functions that use it?